### PR TITLE
[Debug] Remove deprecated ExceptionHandler::createResponse

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: UPGRADE-3.0.md [hunk 1]
```diff
@@ -1259,3 +1259,18 @@ UPGRADE FROM 2.x to 3.0
...
+ * `DebugClassLoader::findFile()` has been removed.
+ * `ErrorHandler::TYPE_DEPRECATION` has been removed.
+ * `ErrorHandler::setLevel()` has been removed, use throwAt() instead.
+ * `ErrorHandler::setDisplayErrors()` has been removed, use throwAt() instead.
+ * `ErrorHandler::setLogger()` has been removed, use setLoggers() or setDefaultLogger() instead.
+ * `ErrorHandler::handle()` has been removed, use handleError() instead.
+ * `ErrorHandler::handleFatal()` has been removed, use handleFatalError() instead.
+ * `ExceptionHandler::createException()` has been removed.
+ * `ExceptionHandler::utf8Htmlize()` has been removed.
+ * `Exception/DummyException` has been removed.
```
Comment (fabpot) [inline UPGRADE-3.0.md]: This should be merged in the branch where it was done instead of master.

Comment (nicolas-grekas) [inline UPGRADE-3.0.md]: this is master, where it has been removed

Comment (fabpot) [inline UPGRADE-3.0.md]: I meant that it should be added in the branch where the deprecations were done.

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**